gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -805,15 +805,13 @@ public GradeSaveResponse saveGrade(final Long assignmentId, final String student
 						// iterate the permissions
 						// if category, compare the category,
 						// then check the group and find the user in the group
-						// if all ok, mark it as GRADEABLE
-
-						boolean categoryOk = false;
-						boolean groupOk = false;
-						boolean gradeable = false;
-
+						// add all available permission functions to the GbGradeInfo
 						for (final PermissionDefinition permission : permissions) {
 							// we know they are all GRADE so no need to check here
```

Comment:
Ah ok.. I think I got it.  Will close this and reopen with a slightly tweaked version of what was there.